app/templates/new-platform-flexberry-g-i-s-map-layer-edit.hbs

```diff
@@ -0,0 +1,56 @@
+<h3 class="ui header">Map layer</h3>
+<form class="ui {{formState}} form flexberry-vertical-form" role="form">
+  <div class="field">
+    <div class="sixteen wide">
+      {{flexberry-validationsummary errors=model.errors}}
+    </div>
+  </div>
+  <div class="field">
+    <div class="flexberry-edit-panel">
+      {{#unless readonly}}
+        {{#unless (and hasParentRoute (not saveBeforeRouteLeave))}}
+          <button type="submit" class="ui positive button" {{action "save"}}>{{t "forms.edit-form.save-button-text"}}</button>
+        {{/unless}}
+        {{#unless (and hasParentRoute (not saveBeforeRouteLeave))}}
+          <button type="submit" class="ui positive button" {{action "saveAndClose"}}>{{t "forms.edit-form.saveAndClose-button-text"}}</button>
+        {{/unless}}
+        {{#unless (and model.isNew (or (not hasParentRoute) (and hasParentRoute saveBeforeRouteLeave)))}}
+          <button type="submit" class="ui negative button" {{action "delete"}}>{{t "forms.edit-form.delete-button-text"}}</button>
+        {{/unless}}
+      {{/unless}}
+      <button type="submit" class="ui button" {{action 'close'}}>{{t "forms.edit-form.close-button-text"}}</button>
+    </div>
+  </div>
+  <div class="field {{if model.errors.name "error" ""}}">
+    {{flexberry-field
+      value=model.name
+      label='Name'
+      readonly=readonly
+    }}
+    {{flexberry-validationmessage error=model.errors.name pointing="pointing"}}
+  </div>
+  <div class="field {{if model.errors.coordinateReferenceSystem "error" ""}}">
+    {{flexberry-field
+      value=model.coordinateReferenceSystem
+      label='CoordinateReferenceSystem'
+      readonly=(or readonly (not model.isNew))
+    }}
+    {{flexberry-validationmessage error=model.errors.coordinateReferenceSystem pointing="pointing"}}
+  </div>
+  <div class="field {{if model.errors.type "error" ""}}">
+    {{flexberry-field
+      value=model.type
+      label='Type'
+      readonly=(or readonly (not model.isNew))
+    }}
+    {{flexberry-validationmessage error=model.errors.type pointing="pointing"}}
+  </div>
+  <div class="ui segment">
+    {{new-platform-flexberry-g-i-s-map-layer-setting
+      value=model.settingsAsObject
+      linkToFieldValue=model.type
+      renderInto='new-platform-flexberry-g-i-s-map-layer-edit'
+      saveValueToFieldName='settings'
+    }}
```

Comment:
I do not understand.
Template of "new-platform-flexberry-g-i-s-map-layer-setting" displays validation inside.
What validation do you wand for model.settingsAsObject?